apps/emails-and-messages/src/components/manage-permissions-text-link.tsx

```diff
@@ -0,0 +1,58 @@
+import { useAppBridge, actions } from "@saleor/app-sdk/app-bridge";
+import { SaleorVersionCompatibilityValidator } from "@saleor/apps-shared";
+import { PermissionEnum } from "../../generated/graphql";
+import { Text } from "@saleor/macaw-ui/next";
+import { TextLink } from "@saleor/apps-ui";
+
+interface ManagePermissionsTextLinkProps {
+  missingPermission: PermissionEnum;
+}
+
+/*
+ * Returns TextLink component with link to manage permissions if used in compatible dashboard version.
+ * Otherwise returns text instructions to reinstall the app.
+ */
+export const ManagePermissionsTextLink = ({
+  missingPermission,
+}: ManagePermissionsTextLinkProps) => {
+  const { appBridgeState, appBridge } = useAppBridge();
+  const dashboardVersion = appBridgeState?.dashboardVersion;
+
+  // Editing app permissions has been introduced in Saleor Dashboard 3.15
+  const isPermissionManagementAvailable = dashboardVersion
+    ? new SaleorVersionCompatibilityValidator(">=3.15").isValid(dashboardVersion)
+    : false;
```

Comment:
**thought:** Not a lot of code, but we should keep an eye on it for a potential extraction to a shared lib. Looks quite reusable.

Reply:
Its is in shared lib ;)

Reply:
`SaleorVersionCompatibilityValidator` is, but I was talking about making it a hook that has `appBridge` in it